src/runtime/auto-prompt-manager.js

```diff
@@ -132,6 +132,7 @@ export class AutoPromptManager {
    *   autoPromptType: (AutoPromptType|undefined),
    *   alwaysShow: (boolean|undefined),
    *   displayLargePromptFn: (function()|undefined),
+   *   isAccessibleForFree: (boolean|undefined),
```

Comment:
Could you update the `showAutoPrompt_` method's comment too? Or possibly create a @typedef ShowAutoPromptParams both could share?

Reply:
Cool, maybe I missed it